targets/pyportal.json

```diff
@@ -3,6 +3,7 @@
     "build-tags": ["pyportal"],
     "flash-1200-bps-reset": "true",
     "flash-method": "msd",
+    "serial-port": ["acm:239a:8035", "acm:239a:35", "acm:239a:8036"],
```

Comment:
Note: `239a:8036` is used by the default firmware on the board. I believe this is a bug in the firmware.